src/certificates/confirm-modal/ConfirmModal.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import { ActionRow, AlertModal, Button } from '@edx/paragon';
+
+const ConfirmModal = ({
```

Comment:
[qiestion]: Can we use an already [existing generic component](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/generic/modal-notification/index.jsx)?